### ISSUE TITLE
Error codes remaining in visual studio, "could not find part of the path"

### ISSUE THREAD

**Issue by robhoward83** [AvaloniaUI/Avalonia#9296]
I'm getting several errors that make no sense.

I have a folder in the assets folder that contains a zip file of files. When I run the program, the program works, it finds the file, stores the files of that zip file in memory. 

However, I'm getting an error: "Could not find part of the path c:\users..." It identifies it in the mainwindow.xaml. Thing is...there is no reference to that file in the main window.xaml.

The problem? This is killing my previewer. It won't work. 

I've checked the csproj file and made sure all the includes and avalonia resource tags are in place. They are. 

I tried adding the folder directly using the "Include" under ItemGroup. Assets\Prompts (my folder name)

The error pops up for "Could not find part of the path c:\Assets\Prompts"

Obviously that's not correct so I deleted the item group and lines that included that. The error is still there. Just won't go away. I've rebuilt the program (builds just fine), closed the solution, reopened...still there. Running the program works just fine. But I cannot get the live previewer to work. Is this a bug? Am I doing something wrong?

**Comment by timunie** [comment]
Probably the previewer us not able to unzip the files. Can you try without the files being zipped?

**Comment by robhoward83** [comment]
No that's not it. I just removed the zip file and rebuilt. Cleaned. I'm still getting this message:

It's saying "Could not find part of path..." but the path IS correct. I'm looking at it on my drive and in visual studio - it lines up perfectly.

**Comment by timunie** [comment]
Can you upload a simple sample to a new GitHub Repo? Then we can double-check what's wrong or if it's a bug. Thx.

**Comment by jackjoblack2022** [comment]
Yes I will do that in a bit and will update the thread here when it's done. Thanks!

**Comment by jackjoblack2022** [comment]
https://github.com/jackjoblack2022/CreativeWritersToolkitApp

I have made the repository public temporarily. Yeah, it's a mess. But the primary issue is in the mainwindow.xaml. At least, that's what the error code is telling me. I'm not using mvvm, though I have that project loaded. Everything is code behind right now. I'm coming from WinForms, so that's what I'm used to and I'm pressed for speed. 

I have tried to fix this using the csproj, adding avalonia resources for the assets\prompts file but it hasn't worked. I'm not sure what the problem is.

**Comment by timunie** [comment]
The designer has no access to files that are only copied if you build the App. So you have two options:

1. Check for Design.IsDesignMode and exist here or create "fake" files
2. Use the assetLoader to load the files instead of the absolute path. See: https://docs.avaloniaui.net/docs/getting-started/assets#loading-assets-from-code

---- 

IMO best option is to use MVVM instead. It has a steep learning curve, but is worth the effort. Some samples can be found here: https://github.com/AvaloniaUI/Avalonia.Samples

I'll convert this issue to a Q&A as I think this is not a bug in Avalonia nor the designer. If you disagree, let me know. If you agree, mark this as the answer.
